gradle/versions.gradle

```diff
@@ -58,10 +58,8 @@ dependencyManagement {
     }
 
     dependencySet(group: 'io.netty', version: '4.1.96.Final') {
```

Comment:
nit: Libp2p has a bit older version `4.1.90` but it seems ok. May upgrade Libp2p on the next release